.github/workflows/add_annotations.yml

```diff
@@ -41,27 +40,27 @@ jobs:
               });
               const fs = require('fs');
               fs.writeFileSync(
-                '${{ github.workspace }}/linter-output.zip',
+                `${process.env.GITHUB_WORKSPACE}/linter-output.zip`,
                 Buffer.from(download.data),
               );
             }
+        env:
```

Comment:
minor nit: Can we put these env variables above the steps declaration?

It's a bit confusing knowing where `LINT_NAME` came from when first inspecting this workflow

Reply:
yeah I agree :+1: can I do that in a followup though? I'm just putting the `env` below the rest of the step because that's what we do in everywhere else in our `.github/workflows`, but I'd be happy to open a followup PR that changes them all to put the `env` closer to the top of the step